pagopa/pn-frontend
# Fix/PN-13071 - Change how SERCQ component is displayed when PEC is enabled (PF and PG)

## Short description
This PR fixes a bug in SERCQ display mode when PEC is enabled as default legal contact for PF and PG

## How to test
Login into PF and PG (as Admin), add a PEC as default legal contact (wait for it to switch in 'verified' status) and check the SERCQ component is displayed as expected following its figma specs.
